app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,31 @@
+<h2>Sign up</h2>
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+      <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+  <div class="field">
+    <%= f.label :bio %><br />
+    <%= f.text_area :bio, autofocus: true, autocomplete: "bio" %>
+  </div>
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Kindly ensure you modify the sign up form to match the wireframe. You items are somewhat aligned to the left, whereas they should be centered.

  Your form | Sneek Peak Design
  :---:|:---:
  ![euw](https://user-images.githubusercontent.com/54848363/216040474-28457743-23c9-4580-968b-5c77745dceb6.jpg) | ![rwo](https://user-images.githubusercontent.com/54848363/216040464-e055a04b-ffe3-4bb5-a456-2771f3ea831b.jpg)